lib/mix/lib/mix/hex.ex

```diff
@@ -22,15 +22,15 @@ defmodule Mix.Hex do
   end
 
   @doc """
-  Returns true if have required Hex, returns false if don't and don't update,
-  if update then exits.
+  Returns `true` if it has the required `Hex`. If an update is performed, it then exits.
+  Otherwise returns `false` without updating anything. 
   """
   @spec ensure_updated?() :: boolean
   def ensure_updated?() do
     cond do
-      !Code.ensure_loaded?(Hex) ->
+      not Code.ensure_loaded?(Hex) ->
```

Comment:
@lexmag by the way.. I learnt this one from one of your code comments on somebody else's code.  so :+1: to you